### ISSUE TITLE
Tensor shape

### ISSUE THREAD

**Issue by 4pygmalion** [nimarb/pytorch_influence_functions#26]
Thank you for your interest in this issue.

I tried to re-engineer pytorch_influence into TensorFlow 2.0 code.

I wonder how is the tensor shape of return from grad_z() ?

I think the return value is list and includes tensors which are (feature, batch), is it right?

This is because the return value from grad z should be inversed hessian (shape:n_feature x n_feature) when deep learning return as follows F : R ^{feature} -> R^{1} 
(hessian is symmatric matrix)

Considering "upweighting Influence function (loss) = grad_z(z_test, theta) ^ {T} * hessian * grad_z(z, theta) ",
the matrix shape must be as follow: (batch. feature) * (feature, feature) * (feature, 1)
